CONTRIBUTING.md

```diff
@@ -85,14 +85,17 @@ Example: `http://localhost:9876/base/test/web-platform-tests/web-animations/anim
     grunt
     # Optional "grunt test" to make sure everything still passes.
     git add -f *.min.js{,.map}
-    git rm .gitignore
+    git rm --ignore-unmatch .gitignore
     git commit -m 'Add build artifacts from '`cat .git/refs/heads/dev`
-    git push HEAD:refs/heads/master
+    git push origin master
```

Comment:
Cool, so maybe for releases, we use a clone, but for code changes, use fork? I'll add a step in the beginning for cloning the repo.